src/persistentSubscription/listPersistentSubscriptionsToStream.ts

```diff
@@ -46,6 +50,12 @@ Client.prototype.listPersistentSubscriptionsToStream = async function (
     options,
   });
 
+  validateField(schemas.streamName, streamName);
+  validateField(
+    schemas.listPersistentSubscriptionsToStreamOptions.optional(),
+    options
+  );
```

Comment:
This pattern feels really C# style, specially if you are gonna be using `Zod`, instead I would expect to validate a DTO

Reply:
You raise a good point. I will take that into consideration.